include/zephyr/bluetooth/bluetooth.h

```diff
@@ -685,6 +685,28 @@ enum {
 	 * @note Requires @ref BT_LE_ADV_OPT_USE_NAME
 	 */
 	BT_LE_ADV_OPT_FORCE_NAME_IN_AD = BIT(18),
+
+	/**
+	 * @brief Use new random address on every connectable advertising
+	 * start.
```

Comment:
```suggestion
	 * @brief Use new random address on every connectable advertising start.
```

Just FYI, the maximum line length is 100

Reply:
I'm following the common pattern here in this file and don't won't to violate it.

Reply:
That's fair, although I wouldn't consider line length a pattern :D